scss/variables/inputs.js

```diff
@@ -0,0 +1,93 @@
+( function( root, factory ) {
+	if ( typeof define === "function" && define.amd ) {
+		define( [ "./chassis" ], factory );
+	} else if ( typeof exports === "object" ) {
+		module.exports = factory( require( "./chassis" ) );
+	} else {
+		root.chassis = factory( root.chassis );
+	}
+}( this, function( chassis ) {
+
+chassis.inputs = {
+	"shadows": {
+		name: "Styles for input shadows",
+		value: {
+			"disabled": "0px 0px 2px 0px rgba(0, 0, 0, 0.12)",
+			"normal": "0px 2px 2px 0px rgba(0, 0, 0, 0.12)",
+			"focus-base": "0px 0px 8px 0px rgba(0, 0, 0, 0.08)",
+			"focus-spread": "0px 8px 8px 0px rgba(0, 0, 0, 0.18)",
+			"error-base": "0px 0px 2px 0px rgba(198,62,54,0.24)",
+			"error-spread": "0px 2px 2px 0px rgba(198,62,54,0.48)"
+		}
+	},
+	"transitions": {
+		name: "Transition animations",
+		value: {
+			"focus": "all .25s",
+			"blur": "all .1s"
```

Comment:
Should we be setting only the properties we want to transition here instead of using "all"? A quick google search gives me the impression it will be better performance wise to call out the properties.